.CI/build_all

```diff
@@ -24,5 +24,5 @@ if [ -z "$GH_TOKEN" ]; then
     rm -rf ./recipes/example
 
     # We just want to build all of the recipes.
-    conda-build-all ./recipes --matrix-condition "numpy >=1.10" "python >=2.7,<3|>=3.4"
+    conda-build-all ./recipes --matrix-condition "numpy >=1.10" "python 3.5.*"
```

Comment:
You should not change the CI configuration. If you want to build only for `Python 3.5` you need to add a skip statement in the recipe.